pkg/smartcontract/manifest/manifest_test.go

```diff
@@ -278,7 +278,7 @@ func TestManifestToStackItem(t *testing.T) {
 			Trusts: WildUint160s{
 				Value: []util.Uint160{{1, 2, 3}},
 			},
-			Extra: "some extra data",
+			Extra: []byte(`even not a json allowed`),
```

Comment:
Not sure it's good.

Reply:
Actually, manifest still should be a valid JSON, so it's not quite true.